### DISCUSSION TITLE
Benefits not received

### DISCUSSION THREAD

**Discussion by PankajKathat** [community/community#143817] [GitHub Education]
### Hi everyone,

How much time it will take to get my student pack benefits 🤔 my request got approved but showing you will receive your benefits soon

**Comment by the-matrixneo** [comment]
did you receive an email from GitHub Education ?

**Comment by the-matrixneo** [comment]
Hello👋 @PankajKathat look at #143785
Well it will help you🚀

**Comment by davevad93** [comment]
Hi @PankajKathat ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @PankajKathat was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
